### ISSUE TITLE
New version breaks with cordova < 8.0

### ISSUE THREAD

**Issue by slschram** [newrelic/newrelic-cordova-plugin#11]
It seems as though the newest version of newrelic breaks with projects running a version of cordova less than 8.0 (we're currently on 7.1.0). Specifically, we're getting a build path error since the app/ folder doesn't exist. Could you allow backwards compatibility for lesser cordova versions or introduce releases so that we can reference a specific release number in our project? Thanks!

**Comment by wilsolutions** [comment]
Indeed, I got the same issue with `cordova 6.5.0`:
```
Installing "newrelic-cordova-plugin" for android
Failed to install 'newrelic-cordova-plugin':Error: ENOENT: no such file or directory, open 'platforms/android/app/build.gradle'
```

**Comment by yjajkiew** [comment]
Same error here with cordova 8.0.0

**Comment by lkonzen-garupa** [comment]
Same here.

**Comment by wlin-incaps** [comment]
I fixed this by forking the repository and editing the android file with the gradle build path. We reported this issue to their dev team a few months ago, and the recommendation was to fix it on our own. Apparently it is fixed in their private repo, but not this public one.

File: hooks/android/android.js
From:
gradleBuildPath: path.join("platforms", "android", "app", "build.gradle"),
To:
gradleBuildPath: path.join("platforms", "android", "build.gradle"),
